db/migrate/20160720075303_remove_project_id_from_projects.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveProjectIdFromProjects < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :projects, :project_id
+  end
+end
```

Comment:
you are still on branch you should not add new migration to fix first. Just change first one, drop rest and recreate db